src/NzbDrone.Core/Download/Clients/rTorrent/RTorrentSettings.cs

```diff
@@ -37,7 +37,7 @@ public RTorrentSettings()
         [FieldDefinition(1, Label = "Port", Type = FieldType.Textbox)]
         public int Port { get; set; }
 
-        [FieldDefinition(2, Label = "Url Base", Type = FieldType.Textbox, Advanced = true, HelpText = "Adds a suffix the rpc url, see http://[host]:[port]/[urlBase], by default this should be RPC2")]
+        [FieldDefinition(2, Label = "Url Base", Type = FieldType.Textbox, HelpText = "Adds a suffix the rpc url, see http(s)://[host]:[port]/[urlBase]")]
```

Comment:
The `see` seems extraneous to this, maybe `example:` would be better?

For rTorrent (because of two different ruTorrent plugins this is pretty much always set)?